NEWS

```diff
@@ -200,22 +250,35 @@ Changed Functionality
 	expression warning in <command line>, line 1: count underflow (1 - 2)
 	18446744073709551615
 
-- The MQTT scripts registering the analyzer and DPD signatures have
-  been moved from the policy folder to base and are loaded by default.
+- The MQTT scripts registering the analyzer and DPD signatures have been moved
+  from the policy folder to base and are loaded by default.
 
 - Notices created for files transferred over multiple connections will now be
   associated with one of the connections rather than none.
 
+- Non-fatal errors when setting up BPF filtering will no longer cause Zeek to
+  exit, but instead will log the error in reporter.log and continue processing.
```

Comment:
There was also something added to `packet_filter.log` ? Maybe I'm misremembering. Maybe not important.

Reply:
You're right, there was. It was just that the error propogated from libpcap appears in packet_filter.log instead of something generic. I'll add that in.